corehq/apps/sms/tasks.py

```diff
@@ -162,8 +162,9 @@ def handle_domain_specific_delays(msg, domain_object, utcnow):
 
 
 def message_is_stale(msg, utcnow):
-    oldest_allowable_datetime = \
-        utcnow - timedelta(hours=settings.SMS_QUEUE_STALE_MESSAGE_DURATION)
+    sms_stale_duration = msg.custom_metadata.get('sms_stale_after', None) if msg.custom_metadata else None
```

Comment:
why do we need to check msg.custom_metadata? It should be always present

Reply:
just confirming that it already present as an integer value when its read?

Reply:
For tests. There were some tests that were failing.
Yeah. it is present as an integer.

Reply:
Can you try fixing those tests instead and see if they are not many?

Reply:
I looked into the tests and they were not straight forward to do. They are having `custom_metadata` but its value is None.
Should I invest time in fixing the tests?

Reply:
Well at least lets give it a try, I am happy to help figure it out if they are not straight forward.

Reply:
Offlined with @mkangia  on it, since the default value of `custom_metadata` in models in `None` so we would be going with the check here.